<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiscoveryWidget</class>
 <widget class="QWidget" name="DiscoveryWidget">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="sitesTable"/>
   </item>
   <item>
    <widget class="QPushButton" name="refreshButton">
     <property name="text">
      <string>تحديث المواقع</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
